changelog/db.changelog-1.1.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet id="1" author="John Watson">
+    <tagDatabase tag="v1.1.0" />
+  </changeSet>
+  <changeSet id="2" author="John Watson">
+    <addColumn tableName="statements">
+      <column name="schemeName" type="VARCHAR(175)">
+        <constraints nullable="false" />
+      </column>
+      <column name="schemeShortName" type="VARCHAR(50)">
+        <constraints nullable="false" />
+      </column>
+      <column name="schemeYear" type="VARCHAR(4)">
+        <constraints nullable="false" />
+      </column>
+      <column name="schemeFrequency" type="VARCHAR(50)" />
```

Comment:
don't think these fields types are consistent w/ other statement dbs

do we want/need to prefix with "scheme"?

can you add a null constraint (either true or false) for this field

Reply:
the fields dnt match the keys coming into this service

Reply:
The types are all strings when they get this far.

I've prefixed with `scheme` to be consistent with the `address` fields.  I also think it makes it clearer that say frequency is related to scheme rather than publishing.

Given the current content of the email just needs business and scheme, I think this structure is fine now.  But if we see more variance in email content between schemes/post payment adjustments etc, then maybe we should send the full statement object from generator and just save it unstructured here to avoid tight coupling between the two and frequent PRs like this.

Reply:
Added constraint